js/station/form.directive.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+(function(){
+  angular
+  .module("songs").directive("songForm", [
+    "SongFactory",
+    "$state",
+    SongFormDirectiveFunction
+  ]);
+
+  function SongFormDirectiveFunction(SongFactory, $state){
+    return {
```

Comment:
It's a good idea to include a `restrict` property, to declare how the directive will be used.